aries_cloudagent/revocation/models/issuer_rev_reg_record.py

```diff
@@ -149,7 +149,7 @@ async def generate_registry(self, context: InjectionContext):
         issuer: BaseIssuer = await context.inject(BaseIssuer)
         tails_hopper_dir = indy_client_dir(join("tails", ".hopper"), create=True)
 
-        LOGGER.debug("create revocation registry with size:", self.max_cred_num)
+        LOGGER.debug("create revocation registry with size:" + str(self.max_cred_num))
```

Comment:
`("create revocation registry with size: %d", self.max_cred_num)` matches the rest of the code base: Using a `+` here matches none of the code base in any case: python string concatenation works fine with `"a " "b"` and looks more natural when you have to break a line to satisfy flake; e.g.,
```
(
    "hello "
    "world"
    "again"
)
```

Reply:
Yeah, you are undoubtedly right. I noticed an error in my console that this 'self.max_cred_num' value could not be stringified, so I added this old-school concoction to get rid of it. It ended up in this PR, but it shouldn't have been included I guess.

Reply:
It's not the end of the world, just noting that there is a convention here. Good catch in any case.